packages/core/docs/changelog.stories.mdx

```diff
@@ -4,6 +4,75 @@ import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
 
 # Changelog
 
+## 4.0.0
+
+4.0 introduces many new components including enhanced alerts, modals and form controls.
+React is now officially supported with our new `@clr/react` package.
+Several bugfixes have been made to improve performance and treeshaking of components.
+
+### React Support
+
+React is now officially supported with our new `@clr/react` package. Follow our
+[React Documentation](./?path=/story/documentation-react--page) to get started.
+
+```bash
+npm install @clr/react
+```
```

Comment:
should we mention `@clr/core` as an optional peer dependency? Or is that implied since this is in the dox for core?

Reply:
Good point! In the package json of the react package we have it listed as a direct dependency but it really should be moved to peer deps. That way as people build on top of core they share the same version in one app. 

```json
{
  "dependencies": {
    "@clr/core": "4.0.0",
    "@clr/react": "4.0.0",
    "@clr/city": "1.1.0",
    "super-cool-lib-using-core": "1.2.3"
  } 
}
```

So in this example app we would have all these deps installed, then `@clr/react` and `super-cool-lib-using-core` would have their `peerDependencies` list `@clr/core`. This will ensure all apps share the same version like angular libraries. I'll get this fixed if this sounds right.

Reply:
Should it remain under `dependencies` for `src/packages/react` for development (since it is a strict dependency for developing react wrappers), and we move it to `peerDependencies` for the `package.json` of the react package when publishing?

Reply:
Updated the package.js script to set core as a peer dependency when building for publish. Local developement its set as a regular dependency.